### ISSUE TITLE
Undefined variable: g:gutentags_options_file

### ISSUE THREAD

**Issue by AntoineD** [ludovicchabant/vim-gutentags#16]
Gutentag,
I've just installed gutentags and I get this error:
```
Error detected while processing function gutentags#setup_gutentags..<SNR>89_update_tags..gutentags#cta
gs#generate:
line   26:
E121: Undefined variable: g:gutentags_options_file
E116: Invalid arguments for function len(g:gutentags_options_file)
E15: Invalid expression: len(g:gutentags_options_file)
Error detected while processing function gutentags#setup_gutentags:
line   45:
E171: Missing :endif
```

The doc does not mention a `g:gutentags_options_file` setting, did I miss something?

**Comment by madssj** [comment]
You missed something, the docs mention it 

Happened for me as well, using NeoBundle. Setting the variable to an empty string seems to silence the error.

There're no references to the variable, but there are checks for it's existence, and the use of something called `g:gutentags_options_file` later in the `ctags.vim` module.

The following will make gutentags not choke:

```
" Silences #16 in gutentag
let g:gutentags_ctags_options_file = ''
let g:gutentags_options_file = ''
```

**Comment by AntoineD** [comment]
Thanks @madssj :-)

I just skimmed the doc and missed that `g:gutentags_ctags_options_file` was mandatory.
That could be stated more clearly at the beginning of the docs.

**Comment by ludovicchabant** [comment]
That variable is *not* mandatory, it was indeed a bug :) Fixed with 09bb38c67430961798ee7eead04d53b453b39886.
